src/components/EditCoursePage/CourseSkills.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Badge } from '@edx/paragon';
+
+class CourseSkills extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      skillNames: props.input.value || [],
+    };
+  }
+
+  render() {
+    const {
+      className,
+      id,
+      input: {
+        name,
+      },
+      label,
+    } = this.props;
+    const {
+      skillNames,
+    } = this.state;
+    return (
+      <div className="form-group mb-2">
+        <div id={id} name={name} tabIndex="-1">{label}</div>
+        {skillNames.map(skill => (
+          <Badge
+            key={skill.id}
+            className={className}
+            variant="light"
+          >
+            { skill }
+          </Badge>
+        ))}
+      </div>
+    );
+  }
+}
+
+CourseSkills.propTypes = {
+  className: PropTypes.string.isRequired,
+  id: PropTypes.string.isRequired,
+  input: PropTypes.shape({
```

Comment:
shouldn't `input` be renamed to `skills` ?

Reply:
This built in prop is directly coming from the parent component. Every field inside a `redux-form` has an `input` prop coming in.